.changeset/modern-paws-mate.md

```diff
@@ -0,0 +1,9 @@
+---
+'@clerk/nextjs': patch
+'@clerk/shared': patch
+'@clerk/astro': patch
+'@clerk/clerk-react': patch
+'@clerk/vue': patch
+---
+
+Update `useAuth` to handle pending sessions as signed-out by default, with opt-out via `useAuth({ treatPendingAsSignedOut: false })` or `<ClerkProvider treatPendingAsSignedOut={false} />`
```

Comment:
I'll spin up different changeset per package since `<ClerkProvider treatPendingAsSignedOut={false} />` is specific to React